core/chains/evm/txmgr/txmgr.go

```diff
@@ -144,6 +148,11 @@ func NewTxm(db *sqlx.DB, ethClient evmclient.Client, cfg Config, keyStore KeySto
 	} else {
 		b.logger.Info("EthTxReaper: Disabled")
 	}
+	if cfg.EvmUseForwarders() {
+		b.fwdMgr = forwarders.NewFwdMgr(db, ethClient, logPoller, lggr, cfg)
+	} else {
+		b.logger.Infof("EvmForwardManager: Disabled")
```

Comment:
Probably this can be moved to Debug level. Also, `Infof` not required. Just `Info` works here.

Reply:
Kept at info to match the others, happy to change this if its diff from others, or change 3 of them.